Casks/mono-mdk-for-visual-studio.rb

```diff
@@ -1,14 +1,14 @@
 cask "mono-mdk-for-visual-studio" do
-  version "6.12.0.122"
-  sha256 "e4b9964477a05474b6a182b0ca08701c7c56beedd4afb7a8f2ac4af5d26fb1fa"
+  version "6.12.0.140"
+  sha256 "db11b7bd12db4050a9bd4b88d01caea2e7d5c40c1c7757b4a97a206900969673"
 
   url "https://download.mono-project.com/archive/#{version.major_minor_patch}/macos-10-universal/MonoFramework-MDK-#{version}.macos10.xamarin.universal.pkg"
   name "Mono"
   desc "Open source implementation of Microsoft's .NET Framework"
   homepage "https://www.mono-project.com/"
 
   livecheck do
-    url "https://www.mono-project.com/download/vs/"
+    url "https://github.com/mono/mono/tags"
     strategy :page_match
     regex(%r{href=.*?/MonoFramework-MDK-(\d+(?:\.\d+)*).macos10.xamarin.universal\.pkg}i)
   end
```

Comment:
This works for livecheck, however I cannot confirm that there would be a package version available from their download location for each Git tag.

```suggestion
  livecheck do
    url "https://github.com/mono/mono"
    strategy :git
    regex(/mono[._-](\d+(?:\.\d+)+)/i)
  end
```

Reply:
GitHub is probably not reliable to determine stable releases. The latest tag is 6.12.0.147, which mono homepage shows as current preview release https://www.mono-project.com/download/preview/

We need a livecheck method that differentiates Preview and Stable releases